lib/routes/sign.js

```diff
@@ -98,8 +99,10 @@ module.exports = function (log, isA, error, signer, db, domain) {
             verifiedEmail: sessionToken.email
           }
         ).then(function(certResult) {
-          var uid = sessionToken.uid.toString('hex')
-          log.activityEvent('account.signed', uid, request)
+          log.activityEvent('account.signed', request, {
+            uid: uid,
+            created_at: sessionToken.accountCreatedAt
```

Comment:
nit: `account_created_at` may be a better name for this field, since it's not obvious from the log message itself what "created at" is referring to.